src/libappimage/CMakeLists.txt

```diff
@@ -70,6 +76,12 @@ install(
     COMPONENT libappimage-dev
 )
 
+install(
```

Comment:
I think we should move the generated headers in some directory with a different, more meaningful name, e.g., `generated-headers/appimage`. `appimage` as a name is ambiguous.

Reply:
done